plugins/modules/ipbase_info.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2023, Dominik Kukacka <dominik.kukacka@gmail.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: ipbase_info
+version_added: 7.0.0
+short_description: Retrieve IP geolocation and other facts of a host's IP address
```

Comment:
Since this relies on some external service, that one should be mentioned in the short description as well.
```suggestion
short_description: Retrieve IP geolocation and other facts of a host's IP address using the ipbase.com API
```